seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/TextReadStrategy.java

```diff
@@ -180,11 +176,7 @@ public void setSeaTunnelRowTypeInfo(SeaTunnelRowType seaTunnelRowType) {
         }
         initFormatter();
         TextDeserializationSchema.Builder builder =
-                TextDeserializationSchema.builder()
-                        .delimiter(fieldDelimiter)
-                        .dateFormatter(dateFormat)
-                        .dateTimeFormatter(datetimeFormat)
-                        .timeFormatter(timeFormat);
+                TextDeserializationSchema.builder().delimiter(fieldDelimiter);
```

Comment:
> @EricJoy2048 This seems to prevent users from customizing the data format

For reading the contents of the file, the format of the custom read is meaningless, because if the user-defined format is not consistent with the date and time format in the file, it will fail to read, so the format must be consistent with the content format in the file under the reading scenario.

For scenarios where a date or time is written to a file, a user-defined format makes sense.So, I am not change the `TextWriteStrategy` code.

Reply:
OK